### ISSUE TITLE
PCM mono audio passthrough?

### ISSUE THREAD

**Issue by tylercheung** [donmelton/other_video_transcoding#135]
Hi,

This is more of a question than a bug, but I'm noticing when I'm transcoding my Criterion blu-ray rips, the older movies w/ PCM mono audio, it doesn't pass through the original audio but encodes it with libaac.   Is this by design or does other_transcoding or ffmpeg doesn't recognize PCM as a pass-through-able format and gamely tries to convert it to AAC?

(also, how good is libaac now vs. the apple encoders?)

Thanks,
Tyler

**Comment by donmelton** [comment]
@tylercheung No, that's by design. PCM audio streams are uncompressed and often huge. In fact, they can be higher in bitrate than the transcoded video, depending on which encoder and ratecontrol method you choose. So, `other-transcode`, by default, will convert these streams to AAC format for mono and stereo and AC-3 for surround. It's always worked that way.

Of course, you can always add `--main-audio 1=original` to your options and pass PCM through directly. But I don't recommend it because that would really be missing the point of making a smaller, more portable version of your media.

I hope that answers your question.

**Comment by donmelton** [comment]
BTW, `libacc` is terrible and no longer used by `ffmpeg`. Instead, `ffmpeg` uses a newer and better AAC encoder by default. It's not as good as the Apple encoder because, really, nothing is. But it's perfectly fine for movies. I use it all the time.

**Comment by donmelton** [comment]
I'm going to assume that I answered this question and close this for now.

**Comment by tylercheung** [comment]
Yes - apologies for the delay in reply, but this does answer my question - thank you very much!
